### PR TITLE
Initial code add

### PR DESCRIPTION
**Pull request by kle**

### CODE REVIEW

File: build-prod-gavle.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+PUSH_P8_PATH=$5
+PROV_PROFILE_PATH=$6
+
+# Build Android
+echo ">>> Building android production-release for Gävle"
+expo ba --config ./app.gavle.json --release-channel prod-gavle --type app-bundle --keystore-path $KEYSTORE_PATH --keystore-alias $KEYSTORE_ALIAS
+
+# Build iOS
+echo ">>> Building iOS production-release for Gävle"
+expo bi --type archive --config ./app.gavle.json --release-channel prod-gavle --team-id N335UMVM8A --dist-p12-path $DIST_P12_PATH --push-id $PUSH_ID --push-p8-path $PUSH_P8_PATH --provisioning-profile-path $PROV_PROFILE_PATH
```
Comment (tonikansala) [inline build-prod-gavle.sh:21]: Is the team-id secret?

File: build-prod-rauma.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+PUSH_P8_PATH=$5
+PROV_PROFILE_PATH=$6
+
+# Build Android
+echo ">>> Building android production-release for Rauma"
+expo ba --config ./app.rauma.json --release-channel prod-rauma --type app-bundle --keystore-path $KEYSTORE_PATH --keystore-alias $KEYSTORE_ALIAS
+
+# Build iOS
+echo ">>> Building iOS production-release for Rauma"
+expo bi --config ./app.rauma.json --release-channel prod-rauma --team-id N335UMVM8A --dist-p12-path $DIST_P12_PATH --push-id $PUSH_ID --push-p8-path $PUSH_P8_PATH --provisioning-profile-path $PROV_PROFILE_PATH
```
Comment (tonikansala) [inline build-prod-rauma.sh:21]: Is the team-id secret?

File: build-qa-gavle.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+PUSH_P8_PATH=$4
+PROV_PROFILE_PATH=$5
+
+# Build Android
+echo ">>> Building android qa-release for Gävle"
+expo ba --config ./app.gavle.json --release-channel qa-gavle --type app-bundle --keystore-path $KEYSTORE_PATH
+
+# Build iOS
+echo ">>> Building iOS qa-release for Gävle"
+expo bi --config ./app.gavle.json --release-channel qa-gavle --team-id N335UMVM8A --dist-p12-path $DIST_P12_PATH --push-id $PUSH_ID --push-p8-path $PUSH_P8_PATH --provisioning-profile-path $PROV_PROFILE_PATH
```
Comment (tonikansala) [inline build-qa-gavle.sh:20]: Team  ID?

File: build-qa-rauma.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+PUSH_P8_PATH=$4
+PROV_PROFILE_PATH=$5
+
+# Build Android
+echo ">>> Building android qa-release for Rauma"
+expo ba --config ./app.rauma.json --release-channel qa-rauma --type app-bundle --keystore-path $KEYSTORE_PATH
+
+# Build iOS
+echo ">>> Building iOS qa-release for Rauma"
+expo bi --config ./app.rauma.json --release-channel qa-rauma --team-id N335UMVM8A --dist-p12-path $DIST_P12_PATH --push-id $PUSH_ID --push-p8-path $PUSH_P8_PATH --provisioning-profile-path $PROV_PROFILE_PATH
```
Comment (tonikansala) [inline build-qa-rauma.sh:20]: Team ID?

**Comment by kle** [comment]
Ok, removed team id and real urn for vis.

**Pull request merged**